Resources/public/views/show_recommendations.html.twig

```diff
@@ -5,13 +5,11 @@
             var recommendationTemplateRenderer = new eZ.RecommendationTemplateRenderer({
                 templateId: '{{ templateId }}',
                 feedbackUrl: '{{ feedbackUrl }}'
-            });
-
-            var recommendationRestClient = new eZ.RecommendationRestClient({
+            }),
+            recommendationRestClient = new eZ.RecommendationRestClient({
```

Comment:
unsure about the js cs, but would have assumed you now needed to introduce indentation, so maybe better to just keep it like it was for better readability.